tests/apps/lims/test_orderform.py

```diff
@@ -19,6 +19,7 @@ def test_parsing_rml_orderform(rml_orderform: str):
     assert sample_data["pool"] == "pool1"
     assert sample_data["application"] == "RMLP10R300"
     assert sample_data["data_analysis"] == str(Pipeline.FLUFFY)
+    assert sample_data["data_delivery"] == str(DataDelivery.NIPT_VIEWER)
```

Comment:
There should be other ways to write a test like this. It seems like we are expecting a certain state of an object. Perhaps we can create that state and compare the instance to that state instead of comparing each and every field.

Reply:
Also if we had a spec for the orderform the spec IS the test and all these tests would not be necessary, right?

Reply:
sure, sounds good